numba/unicode.py

```diff
@@ -570,6 +570,27 @@ def split_whitespace_impl(a, sep=None, maxsplit=-1):
         return split_whitespace_impl
 
 
+@overload_method(types.UnicodeType, 'ljust')
+def unicode_ljust(string, width, fillchar=' '):
+    if not isinstance(width, types.Integer):
```

Comment:
Similar to `width`, it is necessary to assert that `fillchar` is an instance of `types.UnicodeType` and raise a `TypingError` if not, this also needs testing.